4/Dockerfile.rhel7

```diff
@@ -29,13 +35,18 @@ LABEL summary="$SUMMARY" \
       io.k8s.display-name="Node.js $NODEJS_VERSION" \
       io.openshift.expose-services="8080:http" \
       io.openshift.tags="builder,nodejs,nodejs$NODEJS_VERSION" \
+      io.openshift.s2i.scripts-url="image:///usr/libexec/s2i" \
+      io.s2i.scripts-url="image:///usr/libexec/s2i" \
       com.redhat.dev-mode="DEV_MODE:false" \
       com.redhat.deployments-dir="/opt/app-root/src" \
       com.redhat.dev-mode.port="DEBUG_PORT:5858" \
       com.redhat.component="rh-nodejs4-docker" \
       name="rhscl/nodejs-4-rhel7" \
       version="4" \
-      release="1"
+      release="1" \
+      maintainer="SoftwareCollections.org <sclorg@redhat.com>" \
```

Comment:
I dont think we set this label for rhel dockerfiles (since sclorg is centos based afaik).

@hhorak @ficap Do you remember how we decided to tackle this? I cannot seem to find any result of the discussion.

Reply:
please don't overcomplicate this: it's just a contact information